InstantSearch-Core-Offline-Swift.podspec

```diff
@@ -3,7 +3,7 @@ Pod::Spec.new do |s|
     # ------------
     s.name = 'InstantSearch-Core-Offline-Swift'
     s.module_name = 'InstantSearchCore'
-    s.version = '3.2.0'
+    s.version = '3.2.1'
```

Comment:
No need to manually bump the version, we automatically deploy it when we release a new version. Same for Info.plist and and other podspecs